python/PresentationBuilder/slides/DjangoWikiSlide.py

```diff
@@ -59,19 +62,19 @@ def parse(self, markdown):
     # However, RemarkJS does support indented code blocks, but these blocks need to be indented by
     # four spaces. The following preforms this indenting.
     regex = re.compile(r'(^\s*[\*-].*?\n)(```.*?```\s*\n)', re.MULTILINE|re.DOTALL)
-    markdown = regex.sub(self.__indentListNestedCode, markdown)
+    markdown = regex.sub(self.__subIndentListNestedCode, markdown)
 
     # Extract comments
-    markdown = re.sub(r'(?<![^\s.])(\s*\[\]\(\?\?\?\s*(.*?)\))', self.__storeComment, markdown, re.S)
+    markdown = re.sub(r'(?<![^\s.])(\s*\[\]\(\?\?\?\s*(.*?)\))', self.__subStoreComment, markdown, re.DOTALL)
```

Comment:
Again a good place for re.X since I see negative look behind and other hard to read constructs.